config/routes.rb

```diff
@@ -1,3 +1,12 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html\
+
+  root 'welcome#index'
+
+  resource :dashboard, controller: 'user', only: :show do
+    resources :parties, only: [:new, :create]
+  end
+
+  resources :movies, only: [:index, :show]
+  get '/discover', to: 'movies#search'
```

Comment:
Awesome job getting these routes initially started! This is a great platform we can build off of.